### ISSUE TITLE
Project broken since 6.0.0

### ISSUE THREAD

**Issue by EmielW** [wbkd/react-flow#544]
Hey,

Since update `6.0.0` my custom drag functionality is not working anymore. I can't grab a node on the canvas anymore and my transitions/animations inside my custom edges for dropzones are not working anymore as well. For the custom drag, I used `react-dnd`. Do you have any idea what could have caused this?

**Comment by moklick** [comment]
Hey,

this is really strange we only changed the default styles. 
Is everything working when you install v5.11.1?
Do you get any errors?

**Comment by EmielW** [comment]
Thanks for your quick response.

Everything is working fine in `v5.11.1` it's broken since `v.6.0.0`. I don't get any errors but when I start dragging a node via `react-dnd` the drag handler is triggered but I don't see a drag layer anymore and after a couple of seconds the drag handler is cut off automatically.

**Comment by EmielW** [comment]
I linked `react-flow` locally and i found the problem its because 
```jsx 
const nodeTypesParsed = useMemo(() => createNodeTypes(nodeTypes), []);
const edgeTypesParsed = useMemo(() => createEdgeTypes(edgeTypes), []);
```
is now

```jsx
 const nodeTypesParsed = useMemo(() => createNodeTypes(nodeTypes), [nodeTypes]);
 const edgeTypesParsed = useMemo(() => createEdgeTypes(edgeTypes), [edgeTypes]);
```

**Comment by moklick** [comment]
oh no.. why is that a problem in your case? I could change it to the old version, but I thought it would be better when the types are dynamic props.

**Comment by EmielW** [comment]
Not sure what the problem is in my case. Defining the `nodeTypes` and `edgeTypes` outside the component did fix my problem. Not sure what you meant with the `useMemo` can you give me an example so I can try that as well? 😄 

Thanks for your help 😃

**Comment by floofydoug** [comment]
I was also getting this issue when using 6.0.0. downgraded back to 5.9.3 worked for me.

**Comment by moklick** [comment]
Hey @floofydoug 

could the fix that I explained above work for you too?

**Comment by moklick** [comment]
I am afraid the change causes too much trouble. I changed the behaviour to the previous one. Released in 6.0.3
